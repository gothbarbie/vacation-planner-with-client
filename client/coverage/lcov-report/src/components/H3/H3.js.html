<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/H3/H3.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/components/H3</a> H3.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">// @flow
&nbsp;
import * as React from 'react'
import classnames from 'classnames'
&nbsp;
import './H3.css'
&nbsp;
type props = {
  children: React.ChildrenArray&lt;any&gt;,
  light?: boolean,
  dark?: boolean,
}
&nbsp;
const H3 = ({ children, light, dark }: props) =&gt; (
  &lt;h3
    className={classnames('h3', {
      'h3--light': light,
      'h3--dark': dark,
    })}&gt;
    {children}
  &lt;/h3&gt;
)
&nbsp;
export default H3
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Mar 11 2018 12:53:11 GMT+0100 (CET)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
