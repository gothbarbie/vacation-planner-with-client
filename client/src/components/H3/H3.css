.h3 {
  font-weight: 600;
  font-size: 16px;
  color: #4F4F4F;
}
