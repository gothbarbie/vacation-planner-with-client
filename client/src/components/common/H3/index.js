// @flow

import React from 'react'
import classnames from 'classnames'
import type { Element } from 'react'

import './H3.css'

type props = {
  children: Element<any>,
  light?: boolean,
  dark?: boolean,
}

export default ({ children, light, dark }: props) => (
  <h3
    className={classnames('h3', {
      'h3--light': light,
      'h3--dark': dark,
    })}>
    {children}
  </h3>
)
