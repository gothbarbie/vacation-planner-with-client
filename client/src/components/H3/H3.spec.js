import React from 'react'
import { shallow } from 'enzyme'

import H3 from './H3'

describe('components/H3', () => {
  const h3 = shallow(<H3 />)

  it('renders', () => {
    expect(h3).toMatchSnapshot()
  })
})
