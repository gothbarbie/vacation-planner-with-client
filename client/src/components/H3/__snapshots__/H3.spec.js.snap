// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/H3 renders 1`] = `
<h3
  className="h3"
/>
`;
